extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -140,6 +166,26 @@ private function renderInfoboxHero( $data ) {
 	}
 
 	/**
+	 * renders infobox gallery component
+	 *
+	 * @param array $data - array of images
+	 *
+	 * @return string
+	 */
+	private function renderGallery( $data ) {
+		$helper = new PortableInfoboxRenderServiceHelper();
+
+		$galleryData = Array();
+		$hero = array_shift($data);
+		$galleryData['hero'] = $helper->extendImageData( $hero );
```

Comment:
Also - extendImageData - does not heed to be called for other images than first one?